### DISCUSSION TITLE
Sequent Microsystems Relay?

### DISCUSSION THREAD

**Discussion by ghost** [Dan-in-CA/SIP#300] [Q&A]
Hello,

I am hoping to use a Sequent Microsystems Eight Relays 4A/120V 8-Layer Stackable HAT for Raspberry Pi (<https://sequentmicrosystems.com/collections/all-io-cards/products/8-relays-stackable-card-for-raspberry-pi>). It relies on I2C communication from the Raspberry. I have put it atop a RaspberryPi 4. The relays works on self-check. However, I cannot get any of them to turn on using SIP. 

Is anyone familiar with or used this relay board with SIP?

Thanks, Aric

**Comment by Dan-in-CA** [comment]
That looks very interesting and is a good price.
I am not sure of this will work but the SIP PCF857x plugin is a i2C plugin and might help.

**Comment by ghost** [comment]
Thank you very much for pointing out the PCF857x plugin!

I installed it and it gave no errors. I does NOT seem to be working. Not sure how to figure out why. I have tested the relay using the Sequent Microsystems software and it does work. This also means that i2c is on. 

I have to active-low relay as suggested. 

When I scan for devices in the plugin page I find 3: 0x27, 0x25, 0x20.  I have set the '0xAdr' to each of these with no luck. 

8relind tells me that the board Id is '0'.    Changing the SMBus id to 0 does not change anything. In fact, I tried to set the smbus to 0 under 'Advanced (i2c) tools, but it goes back to 1 when hitting 'set'. Thus, I'm not sure if changing that to '0' would solve my problem. 

Any suggestions are greatly appreciated!! The rain is going to stop soon and I'll have to have a working garden controller very soon!

Thanks, Aric

**Comment by ghost** [comment]
Anyone know how to communicate with someone on gitlab? The person who wrote this relay plugin is on gitlab and I cannot for the life of me figure out how to communicate with him. Unfortunately, it is very different than github.

**Reply by Dan-in-CA** [reply]
The author of that plugin in active on the SIP forum.
https://nosack.com/sipforum/index.php
He goes by "Astrogerard".
Dan
